### ISSUE TITLE
Generate static HTML for disabled javascript

### ISSUE THREAD

**Issue by mcrider** [jasonhinkle/phreeze#40]
Does Phreeze have a good way to render routes as full, static pages for when javascript is disabled (for accessibility reasons and so search engines can crawl your site)?  I noticed the Phreeze demo site doesn't work with javascript disabled.  Basically, what I want to implement is http://lostechies.com/derickbailey/2011/09/26/seo-and-accessibility-with-html5-pushstate-part-2-progressive-enhancement-with-backbone-js/ but I'd love for a framework to handle all the mess for me (e.g. sharing templates between the front and back-end).
